lib/ddtrace/span.rb

```diff
@@ -346,8 +346,8 @@ def to_json(*args)
 
     # Return a human readable version of the span
     def pretty_print(q)
-      start_time = (start_time.to_f * 1e9).to_i rescue '-'
```

Comment:
These were always wrong, as `start_time` in `start_time = start_time.to_f` references the newly declared variable, which it was always `nil`. It now references `self.start_time`, the instance value.